horace_core/sqw/PixelData/@PixelDataFileBacked/private/init_.m

```diff
@@ -60,7 +60,7 @@
 elseif isa(init_data,'memmapfile')
     obj.f_accessor_    = init_data;
     % should this all be rewired thoroughly?
-    obj.full_filename_ = init_data.Filename;
+    obj.full_filename  = init_data.Filename;
```

Comment:
This is `Filename` capitalised?

Reply:
I believe, I've modified this to have it as in `lhs` Initial idea was to have it as in `memmapfile`